### PR TITLE
Synchronization schedule details in the process page

### PR DESCRIPTION
**Pull request by 47Balboa**

### CODE REVIEW

File: roda-common/roda-common-data/src/main/java/org/roda/core/data/v2/ip/disposal/DisposalConfirmationState.java [hunk 1]
```diff
@@ -11,5 +11,5 @@
  * @author Miguel Guimarães <mguimaraes@keep.pt>
  */
 public enum DisposalConfirmationState {
-  PENDING, APPROVED, RESTORED, PERMANENTLY_DELETED, EXECUTION_FAILED
+  PENDING, APPROVED, RESTORED, PERMANENTLY_DELETED, EXECUTION_FAILED, SCHEDULED
```
Comment (hmiguim) [inline roda-common/roda-common-data/src/main/java/org/roda/core/data/v2/ip/disposal/DisposalConfirmationState.java:14]: Why SCHEDULED state is on Disposal Confirmation?

**Review by hmiguim — CHANGES_REQUESTED** [review]
Also, attach some screenshots to see the end result

File: roda-common/roda-common-data/src/main/java/org/roda/core/data/v2/ip/disposal/DisposalConfirmationState.java (hunk 1 above)
Comment (47Balboa) [reply roda-common/roda-common-data/src/main/java/org/roda/core/data/v2/ip/disposal/DisposalConfirmationState.java]: Forgot do remove

**Comment by 47Balboa** [comment]
Screenshot from 2022-05-23 10-14-57
image (3)

**Review by hmiguim — CHANGES_REQUESTED** [review]

File: roda-ui/roda-wui/src/main/java/org/roda/wui/server/browse/BrowserServiceImpl.java [hunk 2]
```diff
@@ -1434,4 +1437,15 @@ public void removeLocalConfiguration(LocalInstance localInstance)
     RODAInstance.removeLocalConfiguration(user, localInstance);
   }
 
+  public String getCrontabValue(String locale) {
+    String SYNC_SCHEDULE = RodaCoreFactory.getRodaConfigurationAsString("core.synchronization.scheduleInfo");
```
[the remark this answers is not in the document]
Comment (47Balboa) [reply roda-ui/roda-wui/src/main/java/org/roda/wui/server/browse/BrowserServiceImpl.java]: Ok, done

**Pull request merged**